RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+CreateEntity.swift

```diff
@@ -582,10 +582,15 @@ extension OnLedgerEntitiesClient {
 		let validatorAddress: ValidatorAddress
 		let token: OnLedgerEntity.NonFungibleToken
 		let claimAmount: RETDecimal
-		let reamainingEpochsUntilClaim: Int
+		let reamainingEpochsUntilClaim: Int?
```

Comment:
Optional, if the stake claim NFT is just created, as in the case of unstaking transactions.